UI/SKCanvasElementShowcase/SKCanvasElementShowcase/Presentation/MainPage.xaml

```diff
@@ -0,0 +1,18 @@
+﻿<Page x:Class="SKCanvasElementShowcase.Presentation.MainPage"
+      xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+      xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+      NavigationCacheMode="Required"
+      Background="{ThemeResource BackgroundBrush}">
+    <Grid>
+        <Grid>
```

Comment:
@ramezgerges missing visible bounds here (`<Grid utu:SafeArea.Insets="VisibleBounds">`), the sample is unusable on some Android/iOS emulators because it is under the status bar.

- Android:
   <img width="408" height="633" alt="image" src="https://github.com/user-attachments/assets/18d16708-45a7-4ec1-b7d6-e7030906aeba" />
  
 - iOS:
   <img width="528" height="919" alt="image" src="https://github.com/user-attachments/assets/66595d32-ffaf-42a5-a608-bb6172803a2a" />